common/containers/Tabs/Swap/components/SwapInfoHeaderTitle.tsx

```diff
@@ -29,11 +29,11 @@ export default class SwapInfoHeaderTitle extends PureComponent<SwapInfoHeaderTit
         <div className="col-xs-3">
           <a
             className="SwapInfo-top-logo"
-            href={bityReferralURL}
+            href={shapeshiftReferralURL}
```

Comment:
This should either be a variable based on provider, or we should remove the provider code altogether.